src/pyFAI/integrator/fiber.py

```diff
@@ -156,33 +243,71 @@ def integrate_fiber(self, data,
         result._set_method = res.method
         result._set_compute_engine = res.compute_engine
 
+        if plot_integrated_area:
```

Comment:
What about the SoC ? this is a module doing processing, here you create a dependency on plotting. Users won't be happy because it creates a dependency on matplotlib hence qt ...